documentation/ipi-install/modules/ipi-install-modifying-install-config-for-no-provisioning-network.adoc

```diff
@@ -0,0 +1,25 @@
+// This is included in the following assemblies:
+//
+// ipi-install-configuration-files.adoc
+
+[id='modifying-install-config-for-no-provisioning-network_{context}']
+
+= Modifying the `install-config.yaml` file for no `provisioning` network (optional)
```

Comment:
I think all other headers we use capital O for optional. If we do, lets change.

Reply:
Changed in two other ocurrences to lower case